src/PockyBot.NET.Persistence/Models/PockyUser.cs

```diff
@@ -14,5 +16,12 @@ public class PockyUser
         public string Username { get; set; }
 
         public List<Role> Roles { get; set; }
+        [InverseProperty("sender")]
+        public List<Peg> PegsGiven { get; set; }
+        [InverseProperty("receiver")]
```

Comment:
What is an `InverseProperty`?

Reply:
It's used to refer to the foreign key the pegs are related to. Needed here because there is more than one person foreign key per peg. I might be using it a bit wrong based on [this page](https://www.entityframeworktutorial.net/code-first/inverseproperty-dataannotations-attribute-in-code-first.aspx), thanks for getting me to check that

Reply:
It's used to refer to the property the pegs relate to (via foreign key). I might be using it wrong [based on this](https://www.entityframeworktutorial.net/code-first/inverseproperty-dataannotations-attribute-in-code-first.aspx), will update my implementation.